hubble/resources.py

```diff
@@ -31,3 +44,24 @@ def on_get(self, req, resp):
             }
             for ev in events
         ]
+
+
+class SendUpdate:
+    def on_post(self, req, resp):
+
+        lat = ""
+        long = ""
+
+        with open(req.geoJson) as geoJson:
```

Comment:
You don't need to `open` this on the filesystem. `json.loads()` loads string data.